src/drivers/roboclaw/roboclaw_main.cpp

```diff
@@ -108,8 +108,9 @@ the driver terminates immediately.
 ### Examples
 
 The command to start this driver is:
-
+```
  $ roboclaw start <device> <baud>
```

Comment:
If you remove the space in front of ` $ roboclaw start <device> <baud>` it should also convert to a block.